blockchain/validators/src/config.rs

```diff
@@ -0,0 +1,109 @@
+use std::collections::{BTreeMap, HashMap};
+use std::ffi::OsStr;
+use std::fs;
+use std::path::Path;
+use mc_attest_core::{VerificationReport, serial};
+use mc_blockchain_types::BlockIndex;
+use mc_common::ResponderId;
+use crate::avr_validator::{AvrValidationRecord, AvrValidator};
+use crate::error::ParseError;
+use displaydoc::Display;
+use mc_attest_verifier::Verifier;
+use mc_crypto_keys::Ed25519Public;
+use serde::{Deserialize, Serialize};
+use serde_with::{serde_as, DisplayFromStr};
+use serde_json;
+use toml;
+use TryFrom;
+
+#[serde_as]
+#[derive(Clone, Debug, Display, Default, Deserialize, Eq, PartialEq, Serialize)]
+pub struct AvrConfig {
+    /// AVR Records
+    #[serde(flatten)]
+    #[serde_as(as = "BTreeMap<_, BTreeMap<DisplayFromStr, _>>")]
+    pub avr_records: BTreeMap<ResponderId, BTreeMap<BlockIndex, AvrConfigRecord>>,
```

Comment:
It seems best to represent this config as a list of clauses, e.g. `Vec<AvrConfigRecord>`.